client/reports.go

```diff
@@ -235,6 +260,7 @@ func (c *Client) staticServerSync(gcas GCAServer, gcasKey glow.PublicKey, gcaKey
 	migrationBytes := append(equipmentKey[:], respBuf[540:respLen-(64+72)]...)
 	newGCASigningBytes := append([]byte("EquipmentMigration"), migrationBytes...)
 	if newGCA != blank && !glow.Verify(gcaKey, newGCASigningBytes, newGCASignature) {
+		c.EventLog.Printf("received new GCA from server with invalid signature: %v", err)
```

Comment:
err is `nil` here right?

Reply:
Right I was just cut & pasting the return which was also wrong